storage/src/adb/current/mod.rs

```diff
@@ -76,12 +80,12 @@ pub struct KeyValueProofInfo<K: Array, V: CodecFixed<Cfg = ()>, const N: usize>
 /// Current db with the given `root`.
 fn verify_key_value_proof<H: CHasher, const N: usize>(
     hasher: &mut H,
+    grafting_height: u32,
     proof: &Proof<H::Digest>,
     loc: Location,
     chunk: &[u8; N],
     root: &H::Digest,
     element: &[u8],
```

Comment:
The `element` passed in is always a `Operation` -- should we pass in that type instead of `&[u8]` instead of having the caller do it? It'd consolidate some `.encode()` calls and might make it more clear what `element` represents

Reply:
Changed to accept Codec so we could get rid of the .encode() calls.